python/test/unit/fem/xtest_ghost_mesh_assembly.py

```diff
@@ -6,62 +6,27 @@
 """Unit tests for assembly"""
 
 import pytest
-from petsc4py import PETSc
-
 import ufl
-from dolfinx import MPI, MeshFunction, fem
+from dolfinx import MPI, fem
 from dolfinx.cpp.mesh import GhostMode
 from dolfinx.function import Function, FunctionSpace
 from dolfinx.generation import UnitSquareMesh
-from ufl import Measure, avg, inner
+from petsc4py import PETSc
+from ufl import avg, inner
 
 
 def dx_from_ufl(mesh):
     return ufl.dx
 
 
-def dx_from_measure(mesh):
-    subdomains = MeshFunction("size_t", mesh, mesh.topology.dim, 1)
-    dx = Measure("dx")(subdomain_data=subdomains, domain=mesh)
-    dx = dx(1)
-    return dx
-
-
-def dx_from_measure_and_subdomain(mesh):
-    dx = dx_from_measure(mesh)
-    return dx(1)
-
-
 def ds_from_ufl(mesh):
     return ufl.ds
 
 
-def ds_from_measure(mesh):
```

Comment:
Is this removal temporary or permanent? (not that it matters in the short time period, as the file is currently skipped)

Reply:
What was the point of `_from_measure` which was anyway marking all mesh entities?  I would like to add later tests that do the integration on dx, dS, ds but only on subset of mesh entities.

Reply:
Ok, so a (better) variation of d*_from_measure_and_subdomain on a subset of mesh entities. The point was #536.